urbansearch/clustering/preprocessing.py

```diff
@@ -0,0 +1,40 @@
+#Tokenizes, Stems, removes stop words and counts
+
+import re
+
+from nltk.tokenize import word_tokenize
```

Comment:
`nltk` is should be present in `requirements.txt`. I believe it isn't right now.